### PR TITLE
Scroll to element including navigation bar and banner notifications

### PR DESCRIPTION
**Pull request by sczerwinski-wikia**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java [hunk 1]
```diff
@@ -26,39 +26,31 @@
...
-import org.openqa.selenium.NoSuchElementException;
-import org.openqa.selenium.TimeoutException;
-import org.openqa.selenium.WebElement;
+import org.openqa.selenium.*;
 import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindAll;
 import org.openqa.selenium.support.FindBy;
 import org.openqa.selenium.support.FindBys;
 
+import java.time.Duration;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java]: MINOR Remove this unused import 'java.time.Duration'. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues
* ![MAJOR][MAJOR] 1 major
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [PageObjectLogging.java#L156](https://github.com/Wikia/selenium-tests/blob/e275e876ca72566fe01b038dc61a2258f19f3c9e/src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java#L156): Either log or rethrow this exception. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues
* ![MAJOR][MAJOR] 1 major
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [PageObjectLogging.java#L156](https://github.com/Wikia/selenium-tests/blob/8450031d391289c911112803c99e54be59495d04/src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java#L156): Either log or rethrow this exception. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [PageObjectLogging.java#L156](https://github.com/Wikia/selenium-tests/blob/71a18b23f0f8843a7595f9abfc0c82fdf7302ca8/src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java#L156): Either log or rethrow this exception. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: src/test/java/com/wikia/webdriver/common/core/elemnt/JavascriptActions.java [hunk 2]
```diff
@@ -77,10 +77,25 @@ public void mouseOver(WebElement element) {
...
-        element);
+
+    int offset = getOffset();
+
+    try {
+      return (Boolean) js.executeScript(
+              "return ($(window).scrollTop() + " + offset + " < $(arguments[0]).offset().top) && ($(window).scrollTop() "
+                      + "+ $(window).height() > $(arguments[0]).offset().top + $(arguments[0]).height() + " + offset + ")",
+              element);
+    } catch (Exception e) {
```
Comment (ludwikkazmierczak) [inline src/test/java/com/wikia/webdriver/common/core/elemnt/JavascriptActions.java]: WebDriverException

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/search/crosswikisearch/CrossWikiSearchPageObject.java [hunk 3]
```diff
@@ -4,14 +4,11 @@
 import com.wikia.webdriver.common.logging.PageObjectLogging;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.SearchPageObject;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.wikipage.WikiArticleHomePage;
+import org.openqa.selenium.*;
```
Comment (ludwikkazmierczak) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/search/crosswikisearch/CrossWikiSearchPageObject.java]: no *

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [PageObjectLogging.java#L156](https://github.com/Wikia/selenium-tests/blob/87c240f70688ce90056e5b64c8ccd079cda3ece1/src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java#L156): Either log or rethrow this exception. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [PageObjectLogging.java#L156](https://github.com/Wikia/selenium-tests/blob/a1f82e60329bbe25e0e071bfa9e8bfdf2ada6d53/src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java#L156): Either log or rethrow this exception. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [PageObjectLogging.java#L156](https://github.com/Wikia/selenium-tests/blob/edebbd82bc4eaa16ae73028657046ebdc722caca/src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java#L156): Either log or rethrow this exception. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [PageObjectLogging.java#L156](https://github.com/Wikia/selenium-tests/blob/c7bdea679b764f3b84e0349113f32b36d6fb4284/src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java#L156): Either log or rethrow this exception. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [PageObjectLogging.java#L156](https://github.com/Wikia/selenium-tests/blob/85c89ced5bf72c93267c4cac56592ab71ccf3177/src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java#L156): Either log or rethrow this exception. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Pull request merged**